src/partition/repart.c

```diff
@@ -1346,6 +1348,17 @@ static int partition_read_definition(Partition *p, const char *path) {
         if (r < 0)
                 return r;
 
+        STRV_FOREACH(drop_in_file, drop_in_files) {
+                r = config_parse(NULL, *drop_in_file, NULL,
```

Comment:
nah, no need to hack this up manually, just use `config_parse_many_nulstr`

Reply:
That wouldn't work here right, since we don't use a fixed config file. Shouldn't it be `config_parse_many()` instead?

Reply:
sure